### DISCUSSION TITLE
Formula 1 - accept first match to a list?

### DISCUSSION THREAD

**Discussion by mkjustuk** [Flexget/Flexget#3981] [General]
Formula 1 downloads pose a number of issues for slick handling.  There are normally several downloads per 'episode' over a weekend and series plugin gets confused.  In order to get the earliest option, I match against several RSS feeds in one task so I get one result accepted if multiple feeds have a match.  However, there are several identical results during a few hours, so each time the task runs it can pickup one or all of those 'dupes'.

What would be useful is if I can have the task compare the results to a 'tick-list' of grouped keywords and only accept them if the group of keywords has never been matched previously, reject otherwise.

For example, this weekend I know I want to accept a match containing all of 'miami', 'qualifying' and '2024' but once accepted always reject against those three words appearing together in any future match.

I don't mind adding a long list of the grouped keywords at the beginning of the season if there's a way to do it.

**Comment by mkjustuk** [comment]
To follow up, I've had a play around with entry_list and list_match and been able to limit results so that exact dupes (entry title) are not picked up, which is great.

However, the issue is that unless the title in the entry list is identical, it still gets through.  So, I think I'm halfway there but still the option to get a match on keywords in the entry title rather than the exact match.

**Comment by gazpachoking** [comment]
Perhaps [regexp_list](https://flexget.com/en/Plugins/List/regexp_list) is what you are looking for?
